packages/modal/src/Modal.spec.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import { render, cleanup } from '@testing-library/react';
+import 'jest-dom/extend-expect';
+import Modal from './Modal';
+
+afterAll(cleanup);
+
+describe('packages/Modal', () => {
+  const modalId = 'modal';
+  const modalContent = 'Modal Content';
+
+  describe('when rendered with content', () => {
```

Comment:
```suggestion
  describe('when rendered with content and set as active', () => {
```

Then we can remove the redundant 'when active props is set` in the child test blocks.

Reply:
👍